{"name": "test_InvalidPassword_ErrorMessageShown", "status": "passed", "description": "Login with valid username and invalid password and check that password error message is shown", "attachments": [{"name": "log", "source": "0ee4422b-9cac-4671-8578-9e6241210e5e-attachment.txt", "type": "text/plain"}], "start": 1745223228147, "stop": 1745223228893, "uuid": "74df760b-ca35-46d8-931d-98a230870342", "historyId": "2d49b488a03f042b1f1d0c77c7867d50", "testCaseId": "2d49b488a03f042b1f1d0c77c7867d50", "fullName": "tests.test_login.TestLogin#test_InvalidPassword_ErrorMessageShown", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "User-PC"}, {"name": "thread", "value": "11988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}